provider/datagen/src/uprops/mod.rs

```diff
@@ -20,9 +20,12 @@
 mod bin_uniset;
 #[cfg(feature = "experimental")]
 mod casemapping;
+mod canonical_decompositions;
+pub(crate) mod codepointtrie;
```

Comment:
question: Can we name this module something more specific?

`codepointtrie` doesn't seem specific enough to describe that it is specifically for collator. There is already a sibling module `enum_codepointtrie` in this crate for data exported for enumerated properties as code point tries.

Reply:
Removed this module. (Made a mistake with git, hence the forced update and bad history.)